karoo_gp/test/conftest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import numpy as np
+
+@pytest.fixture
+def rng():
+    return np.random.default_rng(1000)
+
+@pytest.fixture
+def mock_func():
+    def handler(*args, **kwargs):
+        pass
+    return handler
```

Comment:
If you use an actual `unittest.mock.Mock` here, you will be able to make assertions on the function too (e.g. if it got called, with what args, etc.).